Packs/ServiceNow/Scripts/ServiceNowTroubleshoot/SCRIPT_OUTPUT_FOR_DOC_REVIEW.md

```diff
@@ -0,0 +1,15 @@
+### Enabled Instances Health Information 
+
+
+|Instance Name|Last Pull Time|Names of Active Incidents Created 30 days ago|Number of Incidents Pulled in Last Fetch|Query|Size In Bytes|Total Active Incidents Created 30 days ago|
+|---|---|---|---|---|---|---|
+| ServiceNow_v2_instance_1 | 2025-01-29T11:03:45.122073608Z | ServiceNow Incident INC0041801,<br>ServiceNow Incident INC0041816| 5 | stateNOT IN6,7 | 3105 | 10 |
+ 
```

Comment:
Do you want to say what IN6, 7 are or is it obvious?

Reply:
no it's the query from the configuration.